addons/website/static/src/snippets/s_process_steps/options.js

```diff
@@ -175,17 +188,52 @@ options.registry.StepsConnector = options.Class.extend({
      * @param {integer} height
      * @returns {string}
      */
-    _getPath(type, width, height) {
+    _getPath(type, width, height, stepHeightDifference) {
         const hHeight = height / 2;
         switch (type) {
             case 's_process_steps_connector_line': {
-                return `M 0 ${hHeight} L ${width} ${hHeight}`;
+                const verticalPaddingFactor = (Math.abs(stepHeightDifference) / 8);
+                if (stepHeightDifference >= 0) {
+                    return `M 0 ${stepHeightDifference + this.hCurrentStepIconHeight - verticalPaddingFactor} L ${width} ${
+                        this.hNextStepIconHeight + verticalPaddingFactor
+                    }`;
```

Comment:
My linter propose this like you did below:
```suggestion
                    return `M 0 ${
                        stepHeightDifference + this.hCurrentStepIconHeight - verticalPaddingFactor
                    } L ${width} ${this.hNextStepIconHeight + verticalPaddingFactor}`;
```